Add integration page for egps. (#31842)
home-assistant/home-assistant.github.io: source/_integrations/energenie_power_sockets.markdown

---
title: Energenie Power-Sockets
description: Instructions on how to integrate Energenie Power-Strip switches into Home Assistant.
ha_category:
  - Switch
ha_iot_class: Local Polling

> ha_release: '2024.4'

Reviewer: ```suggestion
ha_release: '2024.5'
```

ha_codeowners:
  - '@gnumpi'
ha_domain: energenie_power_sockets
ha_platforms:
  - switch
ha_integration_type: integration
---

> This integration allows you to include [Energenie USB Power-Sockets](https://energenie.com/item.aspx?id=7556&lang=de) into your Home Assistant setup.

Reviewer 2: ```suggestion
The **Energenie Power-Sockets** {% term integration %} allows you to include [Energenie USB Power-Sockets](https://energenie.com/item.aspx?id=7556&lang=de) into your Home Assistant setup.
```

{% include integrations/config_flow.md %}

## Troubleshooting

> Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.

Reviewer 2: These troubleshooting steps, do they apply to all installation types? Or is this specific to a specific type, such as Home Assistant Core installation?

> Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.

Reviewer 2: ```suggestion
If you are running a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation: Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.
```
Maybe we can add a note here?

> 1. find the *vendor_id* and *product_id* of the USB device by calling `lsusb`:

Reviewer 2: ```suggestion
1. Find the *vendor_id* and *product_id* of the USB device by calling `lsusb`:
```

```bash
lsusb

#e.g.: Bus 001 Device 005: ID 04b4:fd15 Cypress Semiconductor Corp. Energenie EG-PMS2
```

> 2. create an udev rule by calling:

Reviewer 2: ```suggestion
2. Create a udev rule by calling:
```

---
title: Energenie Power-Sockets
description: Instructions on how to integrate Energenie Power-Strip switches into Home Assistant.
ha_category:
  - Switch
ha_iot_class: Local Polling
ha_release: '2024.5'
ha_codeowners:
  - '@gnumpi'
ha_domain: energenie_power_sockets
ha_platforms:
  - switch
ha_integration_type: integration
---

The **Energenie Power-Sockets** {% term integration %} allows you to include [Energenie USB Power-Sockets](https://energenie.com/item.aspx?id=7556&lang=de) into your Home Assistant setup.

{% include integrations/config_flow.md %}

## Troubleshooting

> If you are running a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation: Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.

Reviewer: ```suggestion
If you are running a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation: Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating an udev rule.
```

{% details "Creating an udev rule" %}

1. Find the *vendor_id* and *product_id* of the USB device by calling `lsusb`:

```bash
lsusb

#e.g.: Bus 001 Device 005: ID 04b4:fd15 Cypress Semiconductor Corp. Energenie EG-PMS2
```

> 2. Create a udev rule by calling:

Reviewer: ```suggestion
2. Create an udev rule by calling:
```

Reviewer 2: I disagree. Whether you use *a* or *an* depends on the pronunciation, not the spelling. So although it is spelled with a u, it is pronounced like "yudev". See [Merriam Webster](https://www.merriam-webster.com/grammar/is-it-a-or-an) on the topic.
